src/hints/WotHHint.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import unknown from '../assets/hints/WotH_Unknown.png';
+import woth from '../assets/hints/WotH.png';
+import barren from '../assets/hints/barren.png';
+
+class WotHHint extends React.Component {
```

Comment:
Since this also handles barren, wouldn't `AreaHint` or something be more appropriate?